cfgov/v1/wagtail_hooks.py

```diff
@@ -86,9 +87,31 @@ def log_page_deletion(request, page):
 
 @hooks.register('insert_editor_js')
 def editor_js():
-    js_files = [
-        'js/table-block.js',
-    ]
+    js_files = ['js/table-block.js']
+
+    if wagtail.VERSION >= (2, 0):
```

Comment:
Could we have a comment here explaining why we're inserting the hallo js files?

Reply:
You got it!